var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -604,6 +611,9 @@ def url_for_version(self, version):
     depends_on("eigen@3:", when="+machdyn")
     depends_on("py-cython", when="+mliap+python")
     depends_on("py-cython", when="+ml-iap+python")
+    depends_on("py-pip", when="+python", type="build")
+    depends_on("py-wheel", when="+python", type="build")
+    depends_on("py-build", when="+python", type="build")
     depends_on("py-numpy", when="+python")
     depends_on("py-mpi4py", when="+python+mpi")
```

Comment:
Run-time deps aren't documented but I also see imports of matplotlib and pymol. Currently these are listed as build/link deps, which means the package isn't added to PYTHONPATH when loaded. Should these be build/run or build/link/run?

Reply:
Those are dependencies to examples that aren't reachable from the Python package itself. Only the `lammps` package is installed.

Reply:
> Currently these are listed as build/link deps, which means the package isn't added to PYTHONPATH when loaded. Should these be build/run or build/link/run?

Any thoughts on this?

Reply:
Updated to be build/run dependencies.